.devcontainer/docker-compose-local.yaml

```diff
@@ -22,6 +23,7 @@ services:
     ports:
       - 4566:4566
     environment:
+      # https://docs.localstack.cloud/references/configuration/
```

Comment:
This might be a good comment to leave in this readme instead https://github.com/mozilla-ai/lumigator/blob/main/.devcontainer/README.md

Reply:
I can move the list of compose'd image docs to this file, wdyt?